backend/internal/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+const (
+	ServerAddress = "0.0.0.0:3000"
+
+	TeamWhite = 1
+	TeamBlue  = 2
+
+	AttributeTeam   = "team"
+	AttributeAction = "action"
+)
+
+var (
+	WhiteTeamArea = [4]int{20, 21, 23, 25}
+	BlueTeamArea  = [4]int{22, 24, 26, 27}
```

Comment:
maybe add comment, or document in README? 

What do those numbers mean? How does one configure them?

Reply:
We will add comments explaining the constants and variables for now, They will be included in documentation later

Reply:
"later"?

Reply:
The task to create documentation for developers is further in our product backlog

Reply:
In that case, I assume all developers know what those numbers mean and how they're calculated without those docs, right?